packages/web/components/Quest/QuestTile.tsx

```diff
@@ -12,89 +12,100 @@ import {
-        name={quest.guild.name}
+export const QuestTile: React.FC<Props> = ({ quest }) => {
+  const descriptionSummary = quest.description
+    ? `${quest.description.substring(0, 180)} ...`
+    : 'No description available';
+
+  return (
+    <MetaTile>
+      <Box
+        bgImage={`url(${BackgroundImage})`}
+        bgSize="cover"
+        bgPosition="center"
+        position="absolute"
+        top={0}
+        left={0}
+        w="100%"
+        h="3.5rem"
       />
-    </Flex>
+      <Flex justify="center" mb={4}>
+        <Avatar
+          size="lg"
+          src={optimizedImage('logoURL', quest.guild.logo)}
+          name={quest.guild.name}
+        />
+      </Flex>
 
-    <MetaTileHeader>
-      <VStack>
-        <MetaLink as={`/quest/${quest.id}`} href="/quest/[id]">
-          <Heading size="sm" color="white" textAlign="center">
-            {quest.title}
-          </Heading>
-        </MetaLink>
-        <HStack mt={2}>
-          <RepetitionTag
-            repetition={quest.repetition}
-            cooldown={quest.cooldown}
-          />
-          <StatusTag status={quest.status} />
-          <Text>
-            <i>{moment(quest.createdAt).fromNow()}</i>
-          </Text>
-        </HStack>
-      </VStack>
-    </MetaTileHeader>
-    <MetaTileBody flex={1}>
-      <VStack spacing={2} align="stretch">
-        <Box pb={2}>
-          <Text textStyle="caption" pb={1}>
-            DESCRIPTION
-          </Text>
-          <Text>{parse(quest.description || '')}</Text>
-        </Box>
-        <Box pb={2}>
-          <Text textStyle="caption" pb={1}>
-            SKILLS
-          </Text>
-          <SkillsTags
-            skills={quest.quest_skills.map(({ skill }) => skill) as Skill[]}
-            maxSkills={4}
-          />
-        </Box>
-        <Box pb={2}>
-          <Text textStyle="caption" pb={1}>
-            ROLES
-          </Text>
-          {quest.quest_roles.length ? (
-            <RolesTags
-              roles={
-                quest.quest_roles.map(({ PlayerRole: r }) => r) as PlayerRole[]
-              }
+      <MetaTileHeader>
+        <VStack>
+          <MetaLink as={`/quest/${quest.id}`} href="/quest/[id]">
+            <Heading size="sm" color="white" textAlign="center">
+              {quest.title}
+            </Heading>
+          </MetaLink>
+          <HStack mt={2}>
+            <RepetitionTag
+              repetition={quest.repetition}
+              cooldown={quest.cooldown}
+            />
+            <StatusTag status={quest.status} />
+            <Text>
+              <i>{moment(quest.createdAt).fromNow()}</i>
+            </Text>
+          </HStack>
+        </VStack>
+      </MetaTileHeader>
+      <MetaTileBody flex={1}>
+        <VStack spacing={2} align="stretch">
+          <Box pb={2}>
+            <Text textStyle="caption" pb={1}>
+              DESCRIPTION
+            </Text>
+            <Markdown>{descriptionSummary}</Markdown>
```

Comment:
Looks good I'm just wondering if we should render markdown in the tiles.. over at quest chains I've used `remove-markdown` for the summaries when displayed as times. 

imo this makes the view look somewhat consistent, but not always very readable. it's certainly a choice we gotta make.

let's merge for now and then make changes only if necessary.

Reply:
> Looks good I'm just wondering if we should render markdown in the tiles.. over at quest chains I've used `remove-markdown` for the summaries when displayed as times.
> 
> imo this makes the view look somewhat consistent, but not always very readable. it's certainly a choice we gotta make.
> 
> let's merge for now and then make changes only if necessary.

I have more changes to push in about 10 mins. Just testing. Hold off on the merge. @dan13ram

Reply:
![image](https://user-images.githubusercontent.com/780350/194982216-d60bda76-99c7-4a79-93c0-4c604f4c1e00.png)

I've not got the html rendering to the detail page and test to check for html. Will push in a mo.

Reply:
I've also added the correct font for the headings. `Press Start 2P` should have been phased out months ago. Ask @davort 😉

Reply:
code and pre elements in there to test styles.

Reply:
> code and pre elements in there to test styles.

you may want to look at the MarkdownViewer/MarkdownViewer.tsx file to make some specific style changes to the rendered markdown @luxumbra

Reply:
> I've also added the correct font for the headings. `Press Start 2P` should have been phased out months ago. Ask @davort 😉

Lol i know. So we do we want to just remove it from design system in this PR ? xD

Reply:
future improvement thooughts: having two headings basically saying the same thing looks a bit odd to me. thinking the top title should be 'Quest Description' with the quest title in the QuestDetail block as it is now.

Reply:
> > I've also added the correct font for the headings. `Press Start 2P` should have been phased out months ago. Ask @davort wink
> 
> Lol i know. So do we want to just remove it from design system in this PR ? xD

Would be nice but there's a bunch of places where it is still used and globally replacing it will make some font sizes too small as Exo 2 is smaller than Player 2. If this fix wasnt so urgent, i'd say yes but could do it later this week.